tests/nested/manual/core20-to-core22/task.yaml

```diff
@@ -0,0 +1,77 @@
+summary: verify a UC20 to UC22 remodel
+
+# the test may be unstable as UC22 is effectively a work-in-progress thing, and
+# the model in question uses latest/edge of core22 and 22/edge of pc and
+# pc-kernel snaps
+
+systems: [ubuntu-20.04-64]
+
+environment:
+  NESTED_CUSTOM_MODEL: $TESTSLIB/assertions/valid-for-testing-pc-20.model
+  NESTED_IMAGE_ID: uc22-remodel-testing
+  # investigate why system cannot be booted after remodel completes
```

Comment:
what does this comment mean? is this an old comment? or are we not able to boot the system if we use FDE and we do the remodel?

Reply:
I'll make this a TODO, but yes there appears to be a problem when booting with FDE which needs investigation:
![0LjNNyT](https://user-images.githubusercontent.com/41870/144195259-66afaac2-6275-4c76-9589-2fc9a3e4aea3.png)

Reply:
ooooh nice that's a new problem from snap-bootstrap how lovely